cli/core/hack/rpm/tanzu-cli.spec

```diff
@@ -0,0 +1,46 @@
+Name:       tanzu-cli
+Version:    %{cli_version}
+Release:    0
+License:    Apache 2.0
+URL:        https://github.com/vmware-tanzu/tanzu-cli/
+Vendor:     VMware
+Summary:    The Tanzu CLI
+
+%ifarch amd64
+%define arch amd64
+%endif
+
+%ifarch aarch64
+# TODO For now, we use the amd64 build for arm64
+%define arch amd64
```

Comment:
as we dont support arm64 yet ?

Reply:
We don't have an ARM64 asset published on Github yet so I cheat here; I get the AM64 build and it allows Mac M1 to install it and run it in docker.

Once we have an SRP-compliant ARM64 build, we'll do this properly.

Reply:
I see, but Mac users wont be installing the RPMs?

Reply:
You are right, normally Mac users don't use RPM.  It is just that I can test the RPM package using docker on my mac 😄 .  And since we eventually want to support ARM64, this was a good way to prepare.